flink-connectors/flink-connector-hive/pom.xml

```diff
@@ -58,6 +58,28 @@ under the License.
 			<scope>provided</scope>
 		</dependency>
 
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-table-planner-blink_${scala.binary.version}</artifactId>
```

Comment:
Have you discussed about this? IMO the blink planner should not be dependent.

Reply:
ok，we'll remove this dependency and related source interface.